### PR TITLE
Updates License

### PR DESCRIPTION
**Pull request by jfmercer**
Updates License
--

### CODE REVIEW

**Comment by jfmercer** [comment]
The build failed on my machine with the identical error message as on Travis. Like Travis, I used node `0.10.40` and npm `1.4.28`. 

I'll investigate the problem.

**Comment by jfmercer** [comment]
I've relabeled this as "NOMERGE" until the build errors are resolved.

**Comment by jfmercer** [comment]
Because this branch has changed from a simple license update into a debugging operation, I'm going to created a new branch for the debugging commits.

**Comment by jfmercer** [comment]
I've created [a new branch](https://github.com/radify/angular-g11n/tree/101134658-fix-build-error) and a [new PR](https://github.com/radify/angular-g11n/pull/12) to fix the build error. This branch and branch should concern the license file only. Additionally, I moved the two bug-fix commits from this branch to [the new branch](https://github.com/radify/angular-g11n/tree/101134658-fix-build-error), so that the only commits on this branch concern this license.

**Comment by jfmercer** [comment]
Thanks to @gavD's fixes, this is ready to merge.

File: LICENSE [hunk 1]
```diff
@@ -11,7 +11,7 @@ modification, are permitted provided that the following conditions are met:
   this list of conditions and the following disclaimer in the documentation
   and/or other materials provided with the distribution.
 
-* Neither the name of the {organization} nor the names of its
+* Neither the name of angular-g11n nor the names of its
```
Comment (gavD) [inline LICENSE]: organisation should be Radify?

File: karma.conf.js [hunk 2]
```diff
@@ -1,7 +1,27 @@
 module.exports = function(config) {
 	config.set({
+
+    // test results reporter to use
```
Comment (gavD) [inline karma.conf.js]: bad spacing

File: LICENSE (hunk 1 above)
Comment (jfmercer) [inline LICENSE]: We're using the so-called "new", [BSD-3](http://choosealicense.com/licenses/bsd-3-clause/) license. In the template for the BSD-3, `the name of the {organization}` was changed to `the name of [project]`.

**Comment by jfmercer** [comment]
I've fixing the spacing problem in `karma.conf.js`. We should be ready to merge.

**Pull request merged**